pkg/compare/compare_test.go

```diff
@@ -136,6 +141,13 @@ type Checks struct {
 	Err Check
 }
 
+func (c Checks) withPrefixedSuffix(suffixPrefix string) Checks {
```

Comment:
can you add a comment explaining that a it changes the suffix of the **tests expected output files**.
and add that this function can allow having multiple tests with the same ref+resources but with different arguments, without the need to create separate test dir.  

in general just want it to be more easier for new people to  understand.
I think this addition is great and we also can combine some of the test directories that are duplicated due to different arguments (for example i think the yaml\json output)

Reply:
Yeah I though it would be useful for things like that :)

Reply:
Done :)